client/app/ip/ip/reverse/update/translations/Messages_fr_FR.json

```
@@ -0,0 +1,8 @@
+{
+  "ip_table_manage_reverse": "Modifier le reverse",
+  "ip_table_manage_reverse_question": "Insérez un reverse valide pour cette IP, et cliquez sur \"Valider\".",
```

```suggestion
  "ip_table_manage_reverse_question": "Insérez un reverse valide pour cette IP et cliquez sur \"Valider\".",
```

Does it make sense to keep the comma separator?

---

I noticed that you've only moved theses existing translations. But I think the suggestion could be taken in account.

---

I agree it can be removed :)